wranglertools/get_field_info.py

```diff
@@ -40,51 +43,50 @@
     '''
 
 
+def create_common_arg_parser():
+    home = pp.Path.home()
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument('--key',
+                        default='default',
+                        help="The keypair identifier from the keyfile.  \
+                        Default is --key=default")
+    parser.add_argument('--keyfile',
+                        default=f"{home}/keypairs.json",
+                        help=f"The keypair file.  Default is --keyfile={home}/keypairs.json")
+    parser.add_argument('--debug',
+                        default=False,
+                        action='store_true',
+                        help="Print debug messages.  Default is False.")
+    parser.add_argument('--type',
+                        help="To generate a workbook with specific sheets with get_field_info \
+                        or to submit a specified subset of sheets from a multi-sheet workbook with import_data \
+                        specify each sheet by --type",
+                        action="append",
+                        default='all')
```

Comment:
This needs to be a list, since the string `'all'` does not support `action="append"`.
Note that in this case, `get_field_info --type Biosample` results in `types_list = ['all', 'Biosample']`. To fully implement a default here, the function `get_sheet_names` needs to be edited as well, adding logic to remove `'all'` from `types_list` if `len(types_list) > 1`

Reply:
Good catch! I think I've made changes so this now works as expected.